docs/logs/customizing-the-sdk/README.md

```diff
@@ -14,7 +14,14 @@ TODO
 
 ### IncludeScopes
 
-TODO
+A "[scope](https://docs.microsoft.com/dotnet/core/extensions/logging#log-scopes)"
+is an ILogger concept that can group a set of logical operations and attach
+data to each log created as part of a set.
+
+`IncludeScopes` is off by default. Setting this to `true` will include all
+scopes with the exported `LogRecord`.
```

Comment:
Add a note saying that - Consult the individual `Exporter` docs to know more about how each Exporters process scope information.